packages/ui/src/badge/index.ts

```diff
@@ -0,0 +1,2 @@
+export { Badge } from './badge';
+export type { BadgeProps } from './types';
```

Comment:
We should respect [our component guidance](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/README.md#typescript): "This package [...] doesn't export its own types for component props".

We might want to pull that into our README or somewhere else, ideally somewhere that AI agents will respect.

```suggestion
```